node/db/src/parity_db.rs

```diff
@@ -19,24 +19,21 @@ pub struct ParityDb {
     pub db: Arc<parity_db::Db>,
 }
 
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize, PartialEq, Eq)]
 pub struct ParityDbConfig {
-    pub path: PathBuf,
     pub columns: u8,
 }
 
-impl ParityDbConfig {
-    pub fn from_path(path: &Path) -> Self {
-        Self {
-            path: path.to_path_buf(),
-            columns: 1,
-        }
+impl Default for ParityDbConfig {
+    fn default() -> Self {
+        Self { columns: 1 }
     }
 }
 
 impl ParityDb {
     fn to_options(config: &ParityDbConfig) -> Options {
```

Comment:
How about adding the path parameter here?

Reply:
done